sdk/Cargo.toml

```diff
@@ -27,3 +27,6 @@ once_cell = "1.13.0"
 rand = "0.7.3"
 tokio = { version = "1.21.0", features = ["macros", "rt-multi-thread"] }
 url = "2.2.2"
+
+[features]
+cloneable-private-keys = ["aptos-crypto/cloneable-private-keys"]
```

Comment:
Having cloneable private keys is a bad practice, as once you start passing around private keys some bad practices come into play.

Ideally, you'd want to keep it one place and refer to it, so that they can't be used / copied elsewhere.